.gitattributes

```diff
@@ -1,2 +1,12 @@
 *.ipynb linguist-detectable=false
 eia861-transform.ipynb merge=ours
+*.csv text
+*.py text
+*.json text
+*.yml text
+*.yaml text
+*.rst text
+*.toml text
+*.ini text
+*.txt text
+*.md text
```

Comment:
Are there any other "text" file endings that I'm forgetting here?

Reply:
This is a git-specific way of ensuring all of the line endings are standardized to LF (the unix standard) inside the repository, which means that when we edit CSV files with Excel / Libre Office only the lines we change will show up as changed in Git.